packages/common/src/config/env.ts

```diff
@@ -19,5 +20,6 @@ export default Object.keys(process.env)
       'process.env.LOCAL_SERVER': !!LOCAL_SERVER,
       'process.env.STAGING': 'STAGING_BRANCH' in process.env,
       'process.env.VSCODE': !!JSON.stringify(process.env.VSCODE),
+      __DEV__, 
```

Comment:
I'm not sure if this actually worked.

Reply:
I think we need to set it here: https://github.com/CompuIves/codesandbox-client/blob/6508607312aec5c34cfa0aeec480214834ee5169/packages/app/src/sandbox/eval/transpiled-module.ts#L964-L967. It's currently hardcoded but I can make it an option for templates.

One thing we need to make sure of is that the code written in CodeSandbox should also work when you download it to your local environment. Do you know if CRA also has the __DEV__ global enabled?

Reply:
it doesn't. I opened a PR to get it added here https://github.com/facebook/create-react-app/pull/6951

Reply:
After talking to @ide it may seem more logical to make `react-native-web` polyfill this value. Going to roll the change back. After I publish a new version of expo with some better web support, the aliases should be enough to get Expo running in code sandbox.